examples/patterns/grid/nested.html

```diff
@@ -3,29 +3,265 @@
 title: Grid / Nested columns
 category: _patterns
 ---
-<div grid-demo grid-outline>
+
+<div class="p-strip">
   <div class="row">
-    <div class="col-6">
-      <span>.col-6</span>
+    <div class="col-small-4 col-medium-6 col-12">
+      <h2>Nesting at every breakpoint</h2>
+    </div>
+  </div>
+
+  <div class="row" css-grid-demo>
+    <div class="col-small-4 col-medium-6 css-desktop-col-12">
```

Comment:
What is the difference between `col-12` and `css-desktop-col-12`

Reply:
just a left over from when I had to keep shelves and css grid classes different. fixed